x/profiles/keeper/genesis_test.go

```diff
@@ -183,14 +186,15 @@ func (suite *KeeperTestSuite) Test_InitGenesis() {
 	ext := suite.GetRandomProfile()
 
 	testCases := []struct {
-		name      string
-		store     func(ctx sdk.Context)
-		genesis   *types.GenesisState
-		shouldErr bool
-		check     func(ctx sdk.Context)
+		name        string
+		setup       func(ctx sdk.Context)
+		store       func(ctx sdk.Context)
+		genesis     *types.GenesisState
+		shouldPanic bool
```

Comment:
Why was this renamed to `shouldPanic`? I would prefer leaving it as `shouldErr` instead to follow the same naming conventions as other tests

Reply:
Since the failed result of the `InitGenesis` is a panic message, I think the behaviour is different from the error message so I though it was typos.